app/assets/javascripts/invoices.js

```diff
@@ -17,6 +17,10 @@ $(function () {
   $(document).on('change', '.nested-amount', function(e) {
     calculateInvoiceAmount();
   });
+
+  $(document).on('click', '.invoice[data-href]', function(e) {
```

Comment:
@yunixon have you seen event for `[data-edit-url]`? Maybe you can use it, instead of duplicating functionality?